baseapp-auth/baseapp_auth/rest_framework/change_email/serializers.py

```diff
@@ -52,4 +53,8 @@ def update(self, instance, validated_data):
         instance.is_new_email_confirmed = False
         instance.is_email_verified = True
         instance.save()
+        EmailAddress.objects.filter(user=instance, primary=True).update(primary=False)
+        EmailAddress.objects.create(
+            user=instance, email=instance.new_email, verified=True, primary=True
+        )
```

Comment:
Is this a fix for https://silverlogic.atlassian.net/browse/BA-1923? Why not do it in that story?

Reply:
I added in this ticket three stories just to keep the review cycle short